webpack.config.js

```diff
@@ -1,86 +1,88 @@
-
-    chartModulesPath = path.resolve('./src/charts'),
-    fixturesPath = path.resolve('./test/fixtures'),
-    vendorsPath = path.resolve('./node_modules'),
-    bundleIndexPath = path.resolve('./src/bundle.js'),
-
-    projectName = 'britecharts',
-
-    currentCharts = {
-        'bar': './src/charts/bar.js',
-        'donut': './src/charts/donut.js',
-        'legend': './src/charts/legend.js',
-        'line': './src/charts/line.js',
-        'tooltip': './src/charts/tooltip.js',
-        'miniTooltip': './src/charts/mini-tooltip.js',
-        'sparkline': './src/charts/sparkline.js',
-        'stackedArea': './src/charts/stacked-area.js',
-        'stackedBar': './src/charts/stacked-bar.js',
-        'groupedBar': './src/charts/grouped-bar.js',
-        'step': './src/charts/step.js',
-        'brush': './src/charts/brush.js',
-        'loading': ['./src/charts/helpers/load.js'],
-        // hack to make webpack use colors as an entry point while its also a dependency of the charts above
-        'colors': ['./src/charts/helpers/color.js']
-    },
-    currentDemos = {
-        'demo-line': './demos/demo-line.js',
-        'demo-stacked-area': './demos/demo-stacked-area.js',
-        'demo-bar': './demos/demo-bar.js',
-        'demo-grouped-bar': './demos/demo-grouped-bar.js',
-        'demo-stacked-bar': './demos/demo-stacked-bar.js',
-        'demo-donut': './demos/demo-donut.js',
-        'demo-sparkline': './demos/demo-sparkline.js',
-        'demo-step': './demos/demo-step.js',
-        'demo-brush': './demos/demo-brush.js',
-        'demo-kitchen-sink': './demos/demo-kitchen-sink.js'
-    },
-
-    defaultJSLoader = {
-        test: /\.js$/,
-        exclude: /(node_modules)/,
-        use: [{
-            loader: 'babel-loader',
-            options: {
-                presets: ['es2015', 'stage-0'],
-                cacheDirectory: true,
-            }
-        }],
-
-    },
-    babelIstambulLoader = {
-        test: /\.js?$/,
-        include: /src/,
-        exclude: /(node_modules|__tests__|tests_index.js)/,
-        use: [{
-            loader: 'istanbul-instrumenter-loader',
-            options:  {
-                presets: ['es2015', 'stage-0'],
-                cacheDirectory: true,
-            },
-        }],
-    },
-    lintJSLoader = {
-        test: /\.js?$/,
-        include: path.resolve(__dirname, './src/charts'),
-        exclude: /(node_modules)/,
-        enforce: 'pre',
-        loader: 'eslint-loader',
+const webpack = require('webpack');
+const path = require('path');
+const LiveReloadPlugin = require('webpack-livereload-plugin');
+const UglifyJsPlugin = webpack.optimize.UglifyJsPlugin;
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+const merge = require('webpack-merge');
+
+const env = require('yargs').argv.mode;
+const isProduction = env === 'prod' || env === 'prodUMD';
+
+const projectName = 'britecharts';
+
+const CHARTS = {
+    'bar': './src/charts/bar.js',
```

Comment:
Is it necessary for the keys to be strings? Would it be possible to do:
```
const CHARTS = {
    bar: './src/charts/bar.js',
```

Reply:
Yeah, totally

Reply:
Just to clarify, what I am asking is whether it is necessary for the keys to be in quotes.